### ISSUE TITLE
Consider using `figcaption` for caption of concept art on main page

### ISSUE THREAD

**Issue by TimJentzsch** [Leafwing-Studios/leafwing-website#169]
The main page concept art has an image caption.

Here you can consider using the `figure` and `figcaption` elements to explicitly encode this in the DOM ([MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)).

This can improve accessibility as the screenreader then better understands the structure of the page.

**Comment by TimJentzsch** [comment]
FYI I think this `div` here:
![`div` element containing the `img` of the main page](https://user-images.githubusercontent.com/13908946/180517494-ca20080a-7d1e-4c05-b132-5269496d73b5.png)
needs to be a `figure` for this to work properly ([MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)).

I should have probably made this clearer in my issue description.

**Comment by TimJentzsch** [comment]
Also relevant from the [`figcaption` docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption):

> **Permitted parents**: A [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) element; the `<figcaption>` element must be its first or last child.

**Comment by TimJentzsch** [comment]
(Feel free to ignore for now though, at this point I'm just nit-picking :D)

**Comment by alice-i-cecile** [comment]
The nitpicks are a good way to get me to learn the web stack for real :p
